src/votekit/election_state.py

```diff
@@ -114,15 +108,14 @@ def changed_rankings(self) -> Optional[dict]:
         if not self.previous:
             raise ValueError("This is the first round, cannot compare previous ranking")
 
-        else:
-            prev_ranking = self.previous.get_rankings()
-            curr_ranking = self.get_rankings()
-            if curr_ranking == prev_ranking:
-                print("No changes in ranking")
-
-            changes = {}
-            for index, candidate in enumerate(curr_ranking):
-                if candidate != prev_ranking[index]:
-                    prev_rank = prev_ranking.index(candidate)
-                    changes[candidate] = (prev_rank, index)
-            return changes
+        prev_ranking = self.previous.get_rankings()
+        curr_ranking = self.get_rankings()
+        if curr_ranking == prev_ranking:
+            print("No changes in ranking")
+
+        changes = {}
+        for index, candidate in enumerate(curr_ranking):
```

Comment:
this is fine if you're pretty confident number of candidates will remain fairly small let's say N<100, if there's any reason to think it'll get much bigger than that, consider converting both curr/prev to lists of index,candidate so you don't need to do expensive prev.index() lookups as often